modules/installation-azure-service-principal.adoc

```diff
@@ -16,20 +16,17 @@ endif::[]
 [id="installation-azure-service-principal_{context}"]
 = Creating a service principal
 
-Because {product-title} and its installation program must create Microsoft Azure
-resources through Azure Resource Manager, you must create a service principal
-to represent it.
+Because {product-title} and its installation program create Microsoft Azure resources by using the Azure Resource Manager, you must create a service principal to represent it.
 
 .Prerequisites
 
 * Install or update the link:https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-yum?view=azure-cli-latest[Azure CLI].
-* Install the `jq` package.
 * Your Azure account has the required roles for the subscription that you use.
 
 .Procedure
 
 ifdef::ash[]
-. Register your Azure Stack Cloud environment with your Azure CLI. For more details on this process, see Microsoft's documentation for link:https://docs.microsoft.com/en-us/azure-stack/mdc/azure-stack-version-profiles-azurecli-2-tzl#connect-to-azure-stack-hub[Connecting to Azure Stack Hub].
+. Register your Azure Stack Cloud environment with your Azure CLI. For more details on this process, see the link:https://docs.microsoft.com/en-us/azure-stack/mdc/azure-stack-version-profiles-azurecli-2-tzl#connect-to-azure-stack-hub[Microsoft documentation].
 
 .. Register your environment with the Azure CLI:
 +
```

Comment:
@MayXuQQ 
For <environment_name>, should the command specify `AzureStackAdmin`?